server/routes/aws/securityhub_routes.py

```
@@ -0,0 +1,161 @@
+def webhook(org_id: str):
+    """
+    Handle POST requests for AWS Security Hub EventBridge webhooks.
+    Includes API key validation and enqueues background processing.
+    """
+    if request.method == "OPTIONS":
+        return create_cors_response()
+
+    api_key = request.headers.get("x-api-key")
+    if not api_key:
+        EVENTBRIDGE_EVENTS_FAILED.labels(reason="missing_api_key").inc()
+        logger.warning(f"[SECURITY_HUB] Missing x-api-key header for org {org_id}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)